lib/react_on_rails/react_renderer.rb

```diff
@@ -70,7 +71,7 @@ def render_js(js_code, options = {})
       end
     end
 
-    def self.wrap_code_with_exception_handler(js_code, component_name)
+    def self.wrap_code_with_exception_handler(js_code, component_name, prerender)
```

Comment:
Let's not use the variable name `prerender` as that's confusing because what's really important is whether or not the code is running server side.

So how about `server_side`? True means running on server. False means running in client.